tests/app/organisation/test_rest.py

```diff
@@ -840,6 +841,30 @@ def test_get_organisation_services_usage(admin_request, notify_db_session):
     assert service_usage["sms_cost"] == 0.54
 
 
+@pytest.mark.parametrize("num_services", [1, 5, 10])
+@freeze_time("2020-02-24 13:30")
+def test_get_organisation_services_usage_limit_queries_executed(admin_request, notify_db_session, num_services):
+    org = create_organisation(name="Organisation without live services")
+    for _ in range(num_services):
+        service = create_service(service_name=f"service {_}")
+        template = create_template(service=service)
+        dao_add_service_to_organisation(service=service, organisation_id=org.id)
+        create_annual_billing(service_id=service.id, free_sms_fragment_limit=10, financial_year_start=2019)
+        create_ft_billing(
```

Comment:
Should we be creating more rows of billing data for each service to make it more representative of production?

Reply:
I've added a bunch more `ft_billing` entries - is that better?